packages/tools/src/tools/displayTools/Labelmap/labelmapDisplay.ts

```diff
@@ -182,23 +231,25 @@ async function render(
     actorEntry = viewport.getActor(segmentationRepresentationUID);
   }
 
-  const { cfun, ofun } = renderingConfig as LabelmapRenderingConfig;
+  if (actorEntry) {
```

Comment:
how come there will be no actor?, this will not happen since we get the actor if not we create it

Reply:
if the viewport is a StackViewport that could happen

Reply:
we don't have stackViewport Segmentation ....